### PR TITLE
UI updates

### PR DESCRIPTION
**Pull request by ajitzero**
- [x] Better Animations with [aos](https://github.com/michalsnik/aos).
- [x] SEO issues: Domain update, description content, image.
- [x] Performance improvements: preconnects with unpkg & google fonts
- [x] Layout issues for articles and YouTube listings.
- [ ] Update Article Card UI
- [ ] Update YouTube Card UI
- [x] Update License
- [x] Refactor jQuery
- [x] Update experience/about content.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -311,31 +311,25 @@ <h2>Tell me about your project</h2>
 	<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js" integrity="sha384-UO2eT0CpHqdSJQ6hJty5KVphtPhzWj9WO1clHTMGa3JDZwrnQq4sF86dIHNDz0W1" crossorigin="anonymous"></script>
```
Comment (ajitzero) [inline index.html]: - [x] Need to replace links with unpkg.
- [x] `crossorigin="anonymous"`.

File: index.html [hunk 2]
```diff
@@ -311,31 +311,25 @@ <h2>Tell me about your project</h2>
...
+		AOS.init({
+			disable: 'phone',
+			easing: 'ease-in-cubic',
 		});
 
 		$(function () {
 			// Make navbar opaque on scroll
 			var navbar = $('.navbar');
 
 			function setBg() {
```
Comment (ajitzero) [inline index.html:328]: - [x] Replace logic with adding and removing a class.

File: index.html [hunk 3]
```diff
@@ -5,12 +5,12 @@
 	<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
 
 	<!-- TODO OpenGraph tags -->
-	<meta property="og:description" content="">
-	<meta property="og:image" content=""> <!-- Full URL -->
+	<meta property="og:description" content="I'm a creative Android developer based in India, currently working remotely at Small Planet. I'm huge into open-source development and ❤ to contribute back to the community. Interested in working together? Drop me a line!">
+	<meta property="og:image" content="http://ksaitarun.com/assets/profile.jpg">
```
Comment (AjitZero) [inline index.html:9]: - [ ] Use HTTPS

File: index.html [hunk 4]
```diff
@@ -119,42 +118,42 @@ <h1 class="">Hi, I'm Sai&nbsp;Tarun</h1>
...
-					<p>I've previously worked for <strong>Accenture</strong> before moving on to <strong>Small Planet</strong>, where I currently work remotely as an Android Developer, primarily using Flutter for some cool stuff ✨</p>
-					<p class="lead">I'm a Google India &amp; PyTorch Udacity Scholar, and currently under Bertelsmann Scholarship. I'm also a Student Mentor for <strong>Udacity</strong> and contribute actively to several open-source communities such as <strong>AnitaB.org</strong> and <strong>FOSSASIA</strong>.</p>
-					<p class="san-serif">As a student I've completed two internships, one with <strong>Shamgar Software Solutions</strong> and <strong>eBiz Solutions, LLC</strong> I've been a Campus Ambassador for <strong>GeeksForGeeks</strong> as well as for multiple college fests like <strong>Techfest, IIT-B</strong> and <strong>BITS-Pilani, Hyd Campus</strong>, amongst others.</p>
+					<p data-aos="fade-up" data-aos-delay="400">I've previously worked for <strong>Accenture</strong> before moving on to <strong>Small Planet</strong>, where I currently work remotely as an Android Developer, primarily using Flutter for some cool stuff ✨</p>
```
Comment (ajitzero) [inline index.html:134]: @SAITARUN55 Do you want to add info about being a Udacity mentor or anything else. Might have to rethink this section layout.

File: index.html (hunk 1 above)
Comment (ajitzero) [reply index.html]: https://unpkg.com/popper.js@1.14.7/dist/umd/popper.min.js

Comment (ajitzero) [reply index.html]: https://unpkg.com/jquery@3.3.1/dist/jquery.slim.min.js

File: index.html [hunk 5]
```diff
@@ -311,31 +311,25 @@ <h2>Tell me about your project</h2>
...
-		$(document).ready(function(){
-			$('.fade-in').css('opacity', '1');
-			$('.slide-down').css('margin-top', '0');
-			// $('.slide-up').css('margin-top', '0');
-			$('.slide-up').css({
-				marginTop: "0",
-				opacity: "1",
-			});
+		AOS.init({
+			disable: 'phone',
```
Comment (ajitzero) [inline index.html]: Enable for phone as well.

File: index.html (hunk 1 above)
Comment (ajitzero) [reply index.html]: https://unpkg.com/bootstrap@4.3.1/dist/js/bootstrap.min.js

**Pull request merged**